schemas/fin.013.001.01.xsd

```diff
@@ -41,6 +40,7 @@
 	<xs:complexType name="DateAndPlaceOfBirth">
 		<xs:sequence>
 			<xs:element name="BirthDt" type="ISODate"/>
+			<xs:element name="CityOfBirth" type="Max35Text"/>

```

Comment:
This is not needed. FIN schemas need not include all elements from ISO messages, so we cherry pick what we need. City Of Birth is not part of Data Retrieval System response payload.

Reply:
True that, but I chose to add the field for convenience, so the same code can be used to generate both suppl027 and fin013 elements.
Of course, it's not a big deal, but a single bit of field level compatibility. If this is rejected, we have to revert some changes in TR-493, too.